src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/prisonersearch/PagedPrisonerResponse.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.client.prisonersearch
+
+import java.time.LocalDate
+
+data class PagedPrisonerResponse(
```

Comment:
There's some weirdness in the prisoner-search-api swagger spec which prevents us codegen'ing the models as we might otherwise do (some fields in the spec are marked as mandatory, but they are missing in the responses (in dev) so null pointers a-plenty!; plus there was some weird inheritance going on which made the codegen wobble!)

Anyway, given we are only interested in 3 classes, and of those a significant subset of the fields, I decided it was easiest just to create them here.